Stripe/STPPaymentMethodOXXO.m

```diff
@@ -0,0 +1,50 @@
+//
+//  STPPaymentMethodOXXO.m
```

Comment:
Make sure these `.m` files are also in the StripeiOSStatic target

Reply:
Fixed in https://github.com/stripe/stripe-ios/pull/1592/commits/236a98dad6d03f34b03e76ed877799e479e2bd45